src/flows/context/query.tsx

```diff
@@ -74,47 +79,53 @@ const _walk = (node, test, acc = []) => {
   Object.values(node).forEach(val => {
     if (Array.isArray(val)) {
       val.forEach(_val => {
-        _walk(_val, test, acc)
+        find(_val, test, acc)
       })
     } else if (typeof val === 'object') {
-      _walk(val, test, acc)
+      find(val, test, acc)
     }
   })
 
   return acc
 }
 
-export const updateBucketInAST = (ast: File, name: string) => {
-  _walk(
-    ast,
-    node =>
-      node?.type === 'CallExpression' &&
-      node?.callee?.type === 'Identifier' &&
-      node?.callee?.name === 'from' &&
-      node?.arguments[0]?.properties[0]?.key?.name === 'bucket'
-  ).map(
-    node =>
-      (node.arguments[0].properties[0].value.location.source = `"${name}"`)
-  )
-}
+// Removes all instances of nodes that match with the test function
+// and returns the nodes that were returned as an array
+export const remove = (node: File, test, acc = []) => {
```

Comment:
wondering if we should have unit tests for these kinda functions? remove, find, etc.